manifoldco/ui
# plan selector converted to GraphQL

## Reason for change




- Converted the rest of the plan selector to use GraphQL.
- ~Bug: Configurable features are not rendering. That shouldn't take long to fix though.~
- Bug: Cost display is a bit buggy with metered features.
- Bug: Cost not updating to reflect configurable features.

TODO:

## Testing


- Check that all plan selector variations render the same way as the old version.
- Check that events are being emitted and data is accurate when features or region change.
- Check that cost updates correctly when features change.

## Checklist




[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform
